SwiftAppTemplate/SwiftAppTemplate/ReaderListViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ReaderListViewController.swift
+//  SwiftAppTemplate
+//
+//  Created by Catriona Scott on 4/30/19.
+//  Copyright © 2019 Catriona Scott. All rights reserved.
+//
+
+import UIKit
+
+class ReaderListViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    private let tableView = UITableView(frame: CGRect.zero)
+    private weak var cancelButton: UIBarButtonItem?
+
+    private static var cellReuseIdentifier = "readerCell"
+
+    // TODO: update Any to Reader from StripeTerminal
+    private var readers: [Any] = []
+
+    init(readers: [Any]) {
+        super.init(nibName: nil, bundle: nil)
+        tableView.delegate = self
+        tableView.dataSource = self
+        self.readers = readers
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = "Connect Reader"
+
+        view.backgroundColor = UIColor.white
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: ReaderListViewController.cellReuseIdentifier)
+        tableView.separatorStyle = .none
+        view.addSubview(tableView)
+
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+
+        let cancelButton = UIBarButtonItem(title: "Cancel", style: .plain, target: self, action: #selector(dismissAction))
+        self.cancelButton = cancelButton
+        navigationItem.leftBarButtonItem = cancelButton
+
+        tableView.reloadData()
+    }
+
+    @objc func dismissAction() {
+        navigationController?.popViewController(animated: true)
+    }
+
+    // Mark - UITableViewDelegate
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        // TODO: Once a cell has been selected call connectReader on this reader.
+    }
+
+    // Mark - UITableViewDataSource
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return readers.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        let cell = UITableViewCell(style: .default, reuseIdentifier: ReaderListViewController.cellReuseIdentifier)
+
+        // TODO: Set the cell's textLabel to display the reader serial number
```

Comment:
Maybe we can remove this comment, and instead make the default behavior of the ViewController displaying `object.description`?

Reply:
I added a"as?" cast so people could potentially see how to cast to a given object and set the textLabel to different string properties.